libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/InitialGpsEvent.java

```diff
@@ -10,7 +10,7 @@
   private static final String INITIAL_GPS_EVENT_NAME = "initial_gps_event";
 
   InitialGpsEvent(double elapsedTime, String sessionId) {
-    super(sessionId, INITIAL_GPS_EVENT_NAME);
+    super(sessionId, INITIAL_GPS_EVENT_NAME, null);
```

Comment:
Wondering why we aren't capturing the metadata for `InitialGpsEvent` 🤔 Is there any limitation?

Reply:
Yes @Guardiola31337 there is a race condition with the `InitialGpsEventFactory` where it's initialized before `NavigationTelemetry` is initialized. I tried initializing it later but got an error, I don't remember what it was.

Reply:
Could we add a quick `TODO` comment so we don't forget to include the metadata in `InitialGpsEvent`s eventually?